# A service for adding or removing connections from the modem.

# Possible enumerations for connection_type
uint8 TYPE_TCP = 0
uint8 TYPE_UDP = 1

bool add                # TRUE indicates to add the connection, while FALSE indicates to remove the connection.
uint8 connection_type   # The connection type to add/remove
uint16 local_port       # The local port of the connection to add/remove
uint16 remote_port      # The remote port of the connection to add
---
bool success
